romanesco/format/__init__.py

```diff
@@ -248,6 +248,6 @@ def import_default_converters():
     import_converters([os.path.join(cur_path, t) for t in [
         "r", "table", "tree",
         "string", "number", "image",
-        "boolean", "geometry"]])
+        "boolean", "geometry", "netcdf"]])
```

Comment:
Yes there should be a dynamic way to define and import types, even external to the romanesco directory tree.